studentai/titasPetrauskas/lesson-2/class-exercises/4-calculate.js

```diff
@@ -0,0 +1,14 @@
+function calculate(num1, num2, action) {
+  switch (action) {
+    case 'composition': console.log(num1 + num2);
+    break;
+    case 'subtraction': console.log(num1 - num2);
+    break;
+    case 'multiplication': console.log(num1 * num2);
+    break;
+    case 'division': console.log(num1 / num2);
+    break;
+  }
+}
```

Comment:
teisingi atitraukimai atrodytų taip:

```suggestion
function calculate(num1, num2, action) {
  switch (action) {
    case 'composition': 
      console.log(num1 + num2);
      break;
    case 'subtraction': 
      console.log(num1 - num2);
      break;
    case 'multiplication': 
      console.log(num1 * num2);
      break;
    case 'division': 
      console.log(num1 / num2);
      break;
    default:
      throw new Error('Klaida');
  }
}
```